hack/kind/setup.sh

```diff
@@ -77,11 +77,22 @@ setup_cluster() {
   kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout=300s
 }
 
+install_kubectl() {
+    if ! command -v kubectl &> /dev/null; then
+    info "kubectl not found, attempting to install"
+    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
How about using tmp/bin instead of project root so that `git status` and `git add` doesn't include kubectl by mistake?